package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mrshmllw/smores-react",
-  "version": "2.3.4",
+  "version": "2.3.5",
```

Comment:
if you bump in the same PR the release action will probably break

Reply:
Oh i didn't know that... Will remove